### PR TITLE
Fixed latex documentation

### PR DESCRIPTION
**Pull request by aOrionis**
LaTeX expressions didn't look right 
image
Modified the expressions to change that.

### CODE REVIEW

File: src/poliastro/threebody/soi.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 The result is:
 
 .. math::
-    a\\times(\\frac{m}{M}^{\\frac{2}{5}})
+    a\\times(\\frac{m}{M})^{\\frac{2}{5}}
```
Comment (Juanlu001) [inline src/poliastro/threebody/soi.py]: While we are at it, can we remove `\times` and use `\left(` and `\right)` for the delimiters so the correct size is chosen?

File: src/poliastro/threebody/soi.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 The result is:
 
 .. math::
-    a\\times(\\frac{m}{3\\times(M)}^{\\frac{1}{3}})
+    a\\times(\\frac{m}{3\\times(M)})^{\\frac{1}{3}}
```
Comment (Juanlu001) [inline src/poliastro/threebody/soi.py]: There are some useless parentheses around `M` as well.

File: src/poliastro/threebody/soi.py (hunk 1 above)
Comment (Juanlu001) [reply src/poliastro/threebody/soi.py]: Current appearance:

**Comment by aOrionis** [comment]
Are these changes ok?

**Comment by Juanlu001** [comment]
> Are these changes ok?

Absolutely. Thanks again!

**Pull request merged**